aws-cpp-sdk-core/include/aws/core/utils/memory/stl/AWSMap.h

```diff
@@ -13,6 +13,21 @@
 #include <unordered_map>
 #include <cstring>
 
+#ifdef USE_AWS_MEMORY_MANAGEMENT
```

Comment:
Recommend placing this into a header AWSStringHash.h and including it from AWSMap and AWSSet since both use std::hash on Aws::String.